src/main/kotlin/com/autonomousapps/model/Advice.kt

```diff
@@ -49,7 +49,7 @@ data class Advice(
       ofRemove(coordinates, declaration.configurationName)
 
     @JvmStatic
-    fun ofChange(coordinates: Coordinates, fromConfiguration: String, toConfiguration: String) = Advice(
+    fun ofChange(coordinates: Coordinates, fromConfiguration: String?, toConfiguration: String?) = Advice(
```

Comment:
No, please revert this. Change advice _must_ be from non-null to non-null configurations.

Reply:
Indeed, `ofAdd` was not that far 🤦